pyatlan/pkg/templates/package_config.jinja2

```diff
@@ -0,0 +1,87 @@
+from datetime import datetime
+from pathlib import Path
+from pydantic import BaseModel, BaseSettings, Field, parse_obj_as, validator
+from pyatlan.model.assets import Connection
+from pyatlan.model.enums import AtlanConnectorType
+from typing import Any, Optional, Union
+import json
+import logging.config
+import os
+
+PARENT = Path(__file__).parent
+LOGGING_CONF = PARENT / "logging.conf"
+print("LOGGING_CONF.exists():", LOGGING_CONF.exists())
+if LOGGING_CONF.exists():
+    logging.config.fileConfig(LOGGING_CONF)
+LOGGER = logging.getLogger(__name__)
+
+ENV = 'env'
+
+
+def validate_multiselect(v, values, **kwargs):
+    if isinstance(v, str):
+        if v.startswith('['):
+            data = json.loads(v)
+            v = parse_obj_as(list[str], data)
+        else:
+            v = [v]
+    return v
+
+
+def validate_connection(v, values, config, field, **kwargs):
+    v = Connection.parse_raw(v)
+
+
+class ConnectorAndConnection(BaseModel):
+    source: AtlanConnectorType
+    connections: list[str]
+
+
+def validate_connector_and_connection(v, values, config, field, **kwargs):
+    return ConnectorAndConnection.parse_raw(v)
+
+class CustomConfig(BaseModel):
```

Comment:
Is this hard-coding the name of the (generated) custom package-specific config class to `CustomConfig` every time?

Would this be troublesome if we have a custom package that directly calls another custom package — each with their own (different) custom configs both named `CustomConfig`?

Reply:
I'm not sure I'm following you. CustomConfig is the name of the class but it will in it's own .py file i.e. module so it will be namespaced with the module name so there shouldn't be a conflict.

Reply:
I would be interested on more info about a custom package calling another custom package. In that case wouldn't there be a CustomConfig that contains all the parameters for subsequent custom packages?

Reply:
> I'm not sure I'm following you. CustomConfig is the name of the class but it will in it's own .py file i.e. module so it will be namespaced with the module name so there shouldn't be a conflict.

Ah, ok, that'll work.

> I would be interested on more info about a custom package calling another custom package. In that case wouldn't there be a CustomConfig that contains all the parameters for subsequent custom packages?

Yes — so let's say there is one custom package that is going to call 2 others in sequence: there'd be a total of 3 `CustomConfig` classes, but each namespaced separately. So sounds like it should be fine?